ff-parser/src/main/java/org/uniprot/core/flatfile/parser/impl/cc/CcLineModelListener.java

```diff
@@ -714,7 +733,7 @@ public void exitCc_alternative_products(CcLineParser.Cc_alternative_productsCont
                                 .CC_AP_FEATURE_IDENTIFIER();
                 for (TerminalNode terminalNode : terminalNodes) {
                     String text = terminalNode.getText();
-                    name.sequenceFTId.add(text);
+                    name.getSequenceFTId().add(text);
```

Comment:
This is optional, Do you think would be easier to read the code if you create the list, add the items.. and at the end set it to an object?  This is a general comment... I saw it in many other classes as well..

Reply:
I'd say this is more like style of coding. We are not used to it. 
If you replace this
name.getSequenceFTId().add(text);
with
List<String> ftids = new ArrayList<>();
ftids.add(text)
name.setSequenceFtId(ftids);
We may override the existing data in the list. Then we need to find out where we are adding object to the list and add another method like addSequenceFtId(String text) which will be implemented something like
public void addSequenceFtId(String text){
 if(this.sequenceFtid == null){
this.sequenceFtId = new ArrayList<>();
}
this.sequenceFtId.add(text);
}
I am not sure, if such change is needed. It is risky, i think.

Reply:
We have few safe use cases, like the cofactor example

dd.cofactors = new ArrayList<>();	                dd.setCofactors(new ArrayList<>());
                for (Cc_cofactor_lineContext line : lines) {	                for (Cc_cofactor_lineContext line : lines) {
                    CcLineObject.CofactorItem item = new CcLineObject.CofactorItem();	                    CofactorItem item = new CofactorItem();
                    item.name = line.cc_cofactor_name().cc_properties_text_level2().getText();	                    item.setName(line.cc_cofactor_name().cc_properties_text_level2().getText());
                    item.xref = line.cc_cofactor_xref().cc_properties_text_level2().getText();	                    item.setXref(line.cc_cofactor_xref().cc_properties_text_level2().getText());
                    CcLineParser.Cc_cofactor_evidenceContext evidenceContext =	                    CcLineParser.Cc_cofactor_evidenceContext evidenceContext =
                            line.cc_cofactor_evidence();	                            line.cc_cofactor_evidence();
                    if (evidenceContext != null) {	                    if (evidenceContext != null) {
                        EvidenceInfo.processEvidence(	                        EvidenceInfo.processEvidence(
                                object.getEvidenceInfo(), item, evidenceContext.EV_TAG());	                                object.getEvidenceInfo(), item, evidenceContext.EV_TAG());
                    }	                    }
                    dd.cofactors.add(item);	                    dd.getCofactors().add(item);
                }

Reply:
ok. I am doing the simple ones.